### PR TITLE
Create ui-c8y-1021-0-7-fixed-issue-with-info-gauge-resizing-7490.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-61448](https://cumulocity.atlassian.net/browse/MTM-61448)
Original PR: [7490](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7490)

[MTM-61448]: https://cumulocity.atlassian.net/browse/MTM-61448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-7-fixed-issue-with-info-gauge-resizing-7490.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed issue with info gauge resizing. (#7490)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-7-fixed-issue-with-info-gauge-resizing-7490.md]: The Claude AI suggested to use the following title:
```suggestion
title: Fixed info gauge resizing issue
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-7-fixed-issue-with-info-gauge-resizing-7490.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-61448
+version: 1021.0.7
+---
+Fixed issue with info gauge resizing. (#7490)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-7-fixed-issue-with-info-gauge-resizing-7490.md]: The Claude AI suggested to use the following body:
```suggestion
Info gauges in dashboards were not resizing correctly when changing the size of the browser window, leading to a poor user experience. This issue has been fixed and info gauges now resize properly to fit the available space when the browser window size changes. This improvement ensures that dashboards with info gauges remain visually appealing and readable regardless of the user's screen size.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@eniosultan Please check this PR. I recommend to have reviewed and approved this one first and then adapt the one for the 2024 release accordingly.

File: content/change-logs/application-enablement/ui-c8y-1021-0-7-fixed-issue-with-info-gauge-resizing-7490.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed info gauge resizing issue
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-7-fixed-issue-with-info-gauge-resizing-7490.md]:
```suggestion
title: Info gauges resize properly when the window size changes
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**